pkg/controller/service/OWNERS

```diff
@@ -0,0 +1 @@
+reviewers:
```

Comment:
rh-networking please add volunteers to both service and endpoints controller, currently these just contain the networking team members here